Hippo.Core/Domain/Group.cs

```diff
@@ -16,34 +16,36 @@ public class Group
         [MaxLength(250)]
         public string DisplayName { get; set; } = "";
 
-        public bool IsActive { get; set; } = true;
-
         [Required]
         public int ClusterId { get; set; }
         [Required]
         public Cluster Cluster { get; set; }
 
-        public List<Permission> Permissions { get; set; } = new();
+        [JsonIgnore]
+        public List<Account> MemberAccounts { get; set; } = new();
+
+        [JsonIgnore]
+        public List<GroupMemberAccount> MemberAccountsJunction { get; set; } = new();
```

Comment:
What is this `MemberAccountsJunction` and `AdminAccountsJunction` needed?  i don't see it used in mapping or in code anywhere, and it seems like the existing `MemberAccounts` and `AdminAccounts` lists should handle what is needed, right?

Reply:
Oh, yeah I can delete those. Ended up referencing the junction DbSets directly from AppDbContext.